Census/Census.csproj

```diff
@@ -66,6 +65,11 @@
 
   <ItemGroup>
     <Folder Include="Listeners\" />
+    <Folder Include="Migrations\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Shared\Shared.csproj" />
   </ItemGroup>
```

Comment:
Should we include the shared library there instead of the Shared project directly?